client/components/App.jsx

```diff
@@ -2,9 +2,14 @@
 /* eslint-disable no-console */
 import React from 'react';
 import axios from 'axios';
+import styled from 'styled-components';
 import UpperCarousel from './UpperCarousel';
 import LowerCarousel from './LowerCarousel';
 
+const StyledApp = styled.div`
+  background-color: white;
```

Comment:
This is fine, but maybe change it to `#fff` for consistency across your color declarations?